src/shuffle/fwk/gui/ModeIndicator.java

```diff
@@ -106,6 +106,12 @@ private void setup() {
          JLabel label = new JLabel(getString(mode.getI18nKey()));
          label.setFont(modeFont);
          label.setForeground(getColorFor(mode));
+         String modeTooltipKey = "tooltip." + mode.getI18nKey();
+         String modeTooltipText = getString(modeTooltipKey);
+         if (!modeTooltipKey.equals(modeTooltipText)) {
+        	 label.setToolTipText(modeTooltipText);
+         }
+         label.setToolTipText(getString("tooltip."+mode.getI18nKey()));
```

Comment:
Remove line 114, and swap the tab for spaces - 3 spaces per tab is the convention for this project.